source/ftrack_connect_pipeline_qt/ui/assembler/assembler.py

```diff
@@ -662,10 +661,9 @@ def __init__(self, assembler_widget, parent=None):
         :param assembler_widget: :class:`~ftrack_connect_pipeline_qt.ui.assembler.base.AssemblerBaseWidget` instance
         :param parent: The parent dialog or frame
         '''
-        self._asset_widget_class = BrowsedComponentWidget
         self.prev_search_text = None
         super(BrowserListWidget, self).__init__(
-            assembler_widget, parent=parent
+            assembler_widget, BrowsedComponentWidget, parent=parent
```

Comment:
Same as before:
This is a bit confusing. Why are you passing "BrowsedComponentWidget" into the BrowserListWidget init instead of importing it directly on the BrowserListWidget?

Reply:
See comment above